docs/macos/readme.md

```diff
@@ -0,0 +1,155 @@
+# Check for prerequisites
+All requisites should be available for your distribution. The most important are:
+
+* [Git] (https://git-scm.com/downloads)
+* [Docker] (https://docs.docker.com/engine/installation/)
+* [Docker Compose] (https://docs.docker.com/compose/install/)
+
+# Installation
+- Clone and download the project to your working directory:
+
+```
+$ git clone https://github.com/npbtrac/enpii-dev-suite
+```
+
+- Go to the working directory:
+
+```
+$ cd enpii-dev-suite
+```
+
+The current working folder will be `enpii-dev-suite`. After all the prerequisites have been installed, run the following commands to start the installation of docker images.
+  - This command will check if there’s .env file, if there isn’t, it will init and generate an .env file. Then you can repair values that need to be updated located on your based folder.
+
+```
+$ sh scripts/init-env-file.sh
+```
+
+  - When environment variables have been updated, we will generate the docker-compose.yml and configuration files (nginx conf - How to modify nginx conf can be found here #).
+
+```
+$ sh scripts/init-dev-suite.sh
+```
+
+  - Then download and bring up all the containers by the following command:
+
+```
+$ docker-compose up -d --build
+```
+
+- Wait for several mins and you'll have:
+  - Nginx (work as a webserver)
+  - PHP-FPM (PHP execution server)
+  - MySql
+  - Phpmyadmin
+- When it finishes downloading, we can check the list of all services by the command:
+
+```
+$ docker-compose ps
+```
+
+# Configure the nginx conf
+- Firstly, create two files called `index.html` and `index.php` within folder `<path-to-your-root-directorty>` and input the content:
+
+```
+<p> It works </p>
+```
+
+later on, if your web page works, these contents will be shown.
+
+- Before launching the web page, we have to modify the configuration for the `default.conf` file of nginx container, which is located in `etc/nginx`. (Setting can be found here #)
+- Later on, if you want to add configuration for other sites, simply create another conf file for each site and put it into folder `etc/nginx/sites-enabled`.
```

Comment:
This part, you should move it into another sections called "Set up a custom domain" and guide people how to do that: from copy sample `sample-file.conf.example` (there's file in repo) to `your-domain.conf` to repair document root -> how to text nginx configuration and  restart nginx for new config file to take effect